testproject/users/migrations/0005_user_update_profile_name_user_create_profile.py

```
@@ -0,0 +1,43 @@
+                name="create_profile",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="\n            IF NEW.profile_id IS NULL THEN\n                WITH new_profile AS (\n                    INSERT INTO profiles_profile (owner_id, target_content_type_id, target_object_id, name, created, modified, blockers_count, blocking_count, reports_count, comments_count, is_comments_enabled, status)\n                    VALUES (NEW.id, (SELECT id FROM django_content_type WHERE app_label = 'users' AND model = 'user'), NEW.id, TRIM(COALESCE(NEW.first_name || ' ' || NEW.last_name, '')), NOW(), NOW(), 0, 0, '{\"total\":0}'::jsonb, '{\"total\":0,\"main\":0,\"replies\":0,\"pinned\":0,\"reported\":0}'::jsonb, TRUE, 1)\n                    -- Conflicts arise when a Profile for this object already exists\n                    -- (e.g. seeded by a migration or a prior trigger run).  Refresh\n                    -- owner, name, and modified so the profile stays consistent with\n                    -- the current row rather than silently retaining stale values.\n                    ON CONFLICT (target_content_type_id, target_object_id) DO UPDATE\n                        SET owner_id = EXCLUDED.owner_id,\n                            name     = EXCLUDED.name,\n                            modified = EXCLUDED.modified\n                    RETURNING id\n                )\n                UPDATE users_user SET profile_id = (SELECT id FROM new_profile) WHERE id = NEW.id;\n            END IF;\n            RETURN NULL;\n        ",
+                    hash="e8da4b2041d378d80a7ac83687aebcd5240171e3",
+                    operation="INSERT",
+                    pgid="pgtrigger_create_profile_2293f",
+                    table="users_user",
+                    when="AFTER",
+                ),
+            ),
+        ),
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)